modules/nf-core/jvarkit/vcfpolyx/main.nf

```diff
@@ -23,24 +23,28 @@ process JVARKIT_VCFPOLYX {
     task.ext.when == null || task.ext.when
 
     script:
-    def args1  = task.ext.args1 ?: ''
-    def args2  = meta.vcfpolyx_args ?: (task.ext.args2 ?: ' --tag POLYX --max-repeats 10 ')
-    def args3  = task.ext.args3 ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args1         = task.ext.args1 ?: ''
+    def args2         = task.ext.args2 ?: ' --tag POLYX --max-repeats 10 '
```

Comment:
This is very unusual, setting a default `args2`, but then also doing the same in the config file for the test?
Are these essential arguments in some way?

Reply:
those params have been removed.